the-state-of-doctests-in-haskell/article.md

```diff
@@ -0,0 +1,253 @@
+# The state of doctests in Haskell
+
```

Comment:
You need an intro before this heading: it is not good to start an article with a second heading.